lib/src/components/stack/Ancestor_Stack.res

```diff
@@ -1 +1,698 @@
+module Make = (Maker: Ancestor_StylesMaker.T) => {
+  module Base = Ancestor_Base.Make(Maker)
+  module Styles = Ancestor_Styles.Make(Maker)
 
+  %%private(
+    let defaultStyles = Maker.css(`
+      width: 100%;
+      flex-wrap: wrap;
+      display: flex;
+    `)
+
+    let createResponsiveStyles = (currentStyles, value) => {
+      let spacingInRem = value->Maker.unboxBreakpointValue->Styles.Spacing.make
+
+      Styles.mediaQuery(
+        currentStyles,
+        value,
+        `
+          &:before {
+            margin-top: -${spacingInRem};
```

Comment:
Why not use `gap` to create a margin/spacing between the elements? 
I mean, we could use `gap` but to get everything work we should use `flex-direction: column` too.